Frontend/ui-customer/src/Flow.purs

```diff
@@ -310,8 +313,22 @@ currentFlowStatus = do
       else do
         resp <- lift $ lift $ Remote.updateProfile (Remote.mkUpdateProfileRequest)
         pure unit
+      void $ pure $ setCleverTapUserData "Name" ((fromMaybe "" response.firstName) <> " " <> (fromMaybe "" response.middleName) <> " " <> (fromMaybe "" response.lastName))
```

Comment:
@ashkritidewan2000 can you please change the above line to the below written code as if middleName is empty then  two spaces would be added.
```
let middleName = case profile.middleName of 
                    Just ""  -> ""
                    Just name -> (" " <> name)
                    Nothing -> ""
      lastName   = case profile.lastName of 
                    Just "" -> ""
                    Just name -> (" " <> name)
                    Nothing -> ""
      name = (fromMaybe "" profile.firstName) <> middleName <> lastName
```